pkg/apis/keycloak/v1alpha1/keycloak_types.go

```diff
@@ -80,6 +87,15 @@ var (
 	PassthroughTLSTerminationType TLSTerminationType = "passthrough"
 )
 
+type KeycloakExternal struct {
+	// If set to true, this Keycloak will be treated as an external instance.
+	// The unmanaged field also needs to be set to true if this field is true.
+	Enabled bool `json:"enabled,omitempty"`
+	// The URL to use for the keycloak admin API. Needs to be set if external is true.
+	// +optional
```

Comment:
This one is probably required. Otherwise it will fail in the runtime when trying to create a client.

Reply:
Can we set this to required? I didn't find anything about conditional validation. If enabled is false, this field isn't required.

Reply:
ooh I see what you mean. Yeah, good point! Let's leave it as is.